txgithub/api.py

```diff
@@ -40,11 +40,14 @@ class GithubApi(object):
     # - optional user/pass auth (token is not available with v3)
     # - async API
 
-    def __init__(self, oauth2_token, baseURL=None):
+    def __init__(self, oauth2_token, baseURL=None, _reactor=None):
```

Comment:
@tomprince Good deal.  Thank you so much for the review and your general promptness!